docs/streams.html

```diff
@@ -808,6 +808,12 @@
         <h2><a id="streams_upgrade_and_api" href="#streams_upgrade_and_api">Upgrade Guide and API Changes</a></h2>
 
         <p>
+        If you want to upgrade from 0.10.2.x to 0.11.0 you don't need to do any code changes as the API is fully backward compatible.
```

Comment:
I don't feel strongly about including the caveat, but I may expect too much of users -- I like to think the ones that are using internal APIs realize it. You could just put `public` in front of `APIs` and that seems plenty good to me.

For Connect, it is true that 0.11.0.0 is fully compatible with 0.10.2.0.